guides/ReleaseNotes.md

```diff
@@ -1,5 +1,20 @@
 # Release Notes
 
+
+## New Release: [Wix Blog](wix-blog/blog)
+
+The [Blog](wix-blog/blog) API introduces new write functionality:
++ The new [Draft Posts](wix-blog/blog/draft-posts) API allows the ability to create, delete, and manage blog draft posts.
```

Comment:
Hi - I'm not crazy about the use of the word "allow" and it's also a bit repetitive. How about for some of them we say "lets you" or "lets your customers" or "enables" ?